backend/package.json

```diff
@@ -38,7 +38,7 @@
     "@nestjs/config": "3.1.1",
     "@nestjs/core": "10.4.7",
     "@nestjs/jwt": "10.2.0",
-    "@nestjs/platform-express": "10.4.7",
+    "@nestjs/platform-express": "^10.4.15",
```

Comment:
Pin review

Reply:
Updated, but intentionally left the dev dependencies as there is an error on install otherwise (and these won't be installed in production)